assets/components/directDebit/directDebitForm/sortCodeInput.jsx

```diff
@@ -0,0 +1,48 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+type SortCodePropTypes = {
+  sortCode: Array<string>,
+  onChange: Function,
+};
+
+export default function SortCodeInput(props: SortCodePropTypes) {
```

Comment:
Having the export in the middle of the file tends to be messy. I would delete `export default` form this line and added at the bottom of the file as we do in the `directDebitForm.jsx`